api/proto/banyandb/property/v1/rpc.proto

```diff
@@ -96,17 +98,10 @@ service PropertyService {
     option (google.api.http) = {delete: "/v1/property/{metadata.container.group}/{metadata.container.name}/{metadata.id}/{tags}"};
   }
 
-  rpc Get(GetRequest) returns (GetResponse) {
-    option (google.api.http) = {get: "/v1/property/{metadata.container.group}/{metadata.container.name}/{metadata.id}/{tags}"};
-  }
-
-  rpc List(ListRequest) returns (ListResponse) {
+  rpc Query(QueryRequest) returns (QueryResponse) {
```

Comment:
@wu-sheng @wankai123 The new `Query` operation filters the properties. `List` and `Get` will be removed.

Reply:
It is fine.  We will adopt those changes. We prefer querying.